test/helpers/snapshot.ts

```diff
@@ -197,9 +197,8 @@ export async function doSetup(
 
   console.log("deploySnToken");
   const snClaimToken = await deploySnToken(snOwner.address);
-  // TODO: snClaimToken rely, deny
-  // claimToken.rely(radmin);
-  // claimToken.deny(address(this));
+  // await snClaimToken.rely(snOwner.address);
```

Comment:
Shouldn't we uncomment `rely`, `deny` operations?

Reply:
They are not needed as rely is done within the constructor and we directly specify the right owner.